packages/react/src/lib/mergeThemes.ts

```diff
@@ -177,36 +189,29 @@ export const mergeStyles = (...sources: ComponentSlotStyle[]) => {
 }
 
 const mergeThemes = (...themes: ThemeInput[]): ThemePrepared => {
-  const emptyTheme = {
-    siteVariables: {},
-    componentVariables: {},
-    componentStyles: {},
-    fontFaces: [],
-    staticStyles: [],
-    icons: {},
-    animations: {},
-  } as ThemePrepared
-
-  return themes.reduce<ThemePrepared>((acc: ThemePrepared, next: ThemeInput) => {
-    if (!next) return acc
+  return themes.reduce<ThemePrepared>(
+    (acc: ThemePrepared, next: ThemeInput) => {
+      if (!next) return acc
 
-    acc.siteVariables = mergeSiteVariables(acc.siteVariables, next.siteVariables)
+      acc.siteVariables = mergeSiteVariables(acc.siteVariables, next.siteVariables)
 
-    acc.componentVariables = mergeThemeVariables(acc.componentVariables, next.componentVariables)
+      acc.componentVariables = mergeThemeVariables(acc.componentVariables, next.componentVariables)
 
-    acc.componentStyles = mergeThemeStyles(acc.componentStyles, next.componentStyles)
+      acc.componentStyles = mergeThemeStyles(acc.componentStyles, next.componentStyles)
 
-    // Merge icons set, last one wins in case of collisions
-    acc.icons = mergeIcons(acc.icons, next.icons)
+      // Merge icons set, last one wins in case of collisions
+      acc.icons = mergeIcons(acc.icons, next.icons)
 
-    acc.fontFaces = mergeFontFaces(...acc.fontFaces, ...(next.fontFaces || []))
+      acc.fontFaces = mergeFontFaces(...acc.fontFaces, ...(next.fontFaces || []))
 
-    acc.staticStyles = mergeStaticStyles(...acc.staticStyles, ...(next.staticStyles || []))
+      acc.staticStyles = mergeStaticStyles(...acc.staticStyles, ...(next.staticStyles || []))
 
-    acc.animations = mergeAnimations(acc.animations, next.animations)
+      acc.animations = mergeAnimations(acc.animations, next.animations)
 
-    return acc
-  }, emptyTheme)
+      return acc
+    },
+    { ...emptyTheme },
```

Comment:
Clone is required as it will modify the source object. I have two options:
- leave a comment
- use `Object.freeze` on `emptyTheme`

Reply:
leave a comment :)